solutions/18.py

```diff
@@ -0,0 +1,20 @@
+#Himanshu Nachane (@Himanshu1495)
+'''return missing number in an input array of consecutively increasing numbers.
+
+i: [5,6,8,9]
+o: 7
+'''
+
+def missing_num(arr):
+	#get first element of array

```

Comment:
don't put comments for obvious things. The goal should always to write your code in a way so that it doesn't require comments. For example, `firstElement = arr[0]`